client/components/sidebar.vue

```diff
@@ -207,8 +303,40 @@
     padding-right: 0;
   }
 
+  .fade-enter {
+    opacity: 0;
+  }
+
+  .fade-enter-active {
+    transition: opacity 0.5s;
+  }
+
+  .fade-leave-active {
+    transition: opacity 0.2s;
+    opacity: 0;
+  }
+
   .doc_status__label {
     text-transform: uppercase;
     font-weight: bold;
   }
+
+  .reply{
```

Comment:
Please use same style as elsewhere. We have space before `{`.

Moreover, we have a scheme how to name classes. See
* https://github.com/peer/doc/blob/master/CODE_STYLE.md#scss
* https://github.com/airbnb/css#oocss-and-bem

So this should `sidebar__reply`.